src/app/static/src/app/components/genericChart/GenericChart.vue

```diff
@@ -60,12 +60,12 @@
                 </div>
                 <div v-for="dataSource in chartConfigValues.dataSourceConfigValues.filter(ds => ds.tableConfig)"
                      :key="dataSource.config.id">
-                    <download-indicator :filtered-data="filteredDataWithoutPages ? filteredDataWithoutPages[dataSource.config.id] : null"
-                                        :unfiltered-data="unfilteredData ? unfilteredData[dataSource.config.id] : []"></download-indicator>
-                    <generic-chart-table :table-config="dataSource.tableConfig || { columns: [] }"
+                    <download-indicator :filtered-data="filteredDataWithoutPages![dataSource.config.id]"
+                                        :unfiltered-data="unfilteredData[dataSource.config.id]"></download-indicator>
+                    <generic-chart-table :table-config="dataSource.tableConfig!"
                                          :filtered-data="chartData[dataSource.config.id]"
-                                         :columns="dataSource.columns || []"
-                                         :selected-filter-options="dataSource.selections.selectedFilterOptions || {}"
+                                         :columns="dataSource.columns!"
+                                         :selected-filter-options="dataSource.selections.selectedFilterOptions!"
```

Comment:
just reverting this section back to how it actually was in hint dw too much about this